dev/EVAndPATH/Microsoft.Process.Environment/Microsoft.Process.Environment.EnvironmentManager.cpp

```diff
@@ -0,0 +1,51 @@
+﻿#include "pch.h"
+#include "Microsoft.Process.Environment.EnvironmentManager.h"
+#include "EnvironmentManager.g.cpp"
+
+namespace winrt::Microsoft::Process::Environment::implementation
+{
+
+    EnvironmentManager::EnvironmentManager(Scope const& scope)
+            : m_Scope(scope) { }
+
+    Microsoft::Process::Environment::EnvironmentManager EnvironmentManager::GetForProcess()
+    {
+        throw hresult_not_implemented();
```

Comment:
@jonwis @EHO-makai @BenJKuhn @jevansaks 

What's our checkin policy?

Last time we spoke (summer'2020) it was "don't check anything incomplete into main" and specifically cited no incomplete implementation, tests are required and "don't check anything into main you're not ready to receive bug reports from the world".

That held up the Dynamic Dependencies PR for months(!) as API skeleton (like this) per API Review'd and approved spec didn't meet that criteria.

Has this policy changed?

Are the current rules doc'd? I can't find anything on point.

Reply:
This PR is not into master, it's into a feature branch. (The source/target branch names should be namespaced like feature/EVandPATH and user/dhoehna/IDLAndProjectSkeleton to make it more clear)

Reply:
> feature/EVandPATH and user/dhoehna/IDLAndProjectSkeleton

Sorry about that.  I'll change my source branch for the next PR.